container-accesslogging/src/main/java/com/yahoo/container/logging/JSONFormatter.java

```diff
@@ -179,14 +178,8 @@ private BigDecimal durationAsSeconds(long timeInMillis) {
         return duration.setScale(3, BigDecimal.ROUND_HALF_UP);
     }
 
-    private String getNormalizedURI(URI uri) {
-        URI normalizedURI = uri.normalize();
-        String uriString = normalizedURI.getPath();
-        if (normalizedURI.getRawQuery() != null) {
-            uriString = uriString + "?" + normalizedURI.getRawQuery();
-        }
-
-        return uriString;
+    private static String getNormalizedURI(String rawPath, String rawQuery) {
+        return rawQuery != null ? rawPath + "?" + rawQuery : rawPath;
```

Comment:
Maybe extract a package-private method for this, and use it in VespaAccessLog as well?